src/assets/wise5/components/dialogGuidance/dialog-guidance-student/dialog-guidance-student.component.ts

```diff
@@ -210,7 +210,8 @@ export class DialogGuidanceStudentComponent extends ComponentStudent {
       .pipe(timeout(this.cRaterTimeout))
       .subscribe(
         (response: CRaterResponse) => {
-          this.cRaterSuccessResponse(Object.assign(new CRaterResponse(), response));
+          const data = this.CRaterService.getDataFromResponse(response);
+          this.cRaterSuccessResponse(Object.assign(new CRaterResponse(), data));
```

Comment:
Can we change this.CRaterService.getDataFromResponse() to return type CRaterResponse so we don't need to call Object.assign() here?

```suggestion
          this.cRaterSuccessResponse(this.CRaterService.getDataFromResponse(response));
```